### PR TITLE
feature/ S3 Completion

### PR DESCRIPTION
**Pull request by migmruiz**

### CODE REVIEW

File: src/test/scala/com/taxis99/aws/S3ClientSpec.scala [hunk 1]
```diff
@@ -1,10 +1,13 @@
 package com.taxis99.aws
 
+import java.io.ByteArrayInputStream
```
Comment (migmruiz) [inline src/test/scala/com/taxis99/aws/S3ClientSpec.scala]: Codacy Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/99taxis/awsscala/file/2387542323/issues/source?bid=3170898&fileBranchId=3228566#l3)

File: src/main/scala/com/taxis99/aws/S3Client.scala [hunk 2]
```diff
@@ -1,9 +1,12 @@
 package com.taxis99.aws
 
+import java.io.ByteArrayInputStream
```
Comment (migmruiz) [inline src/main/scala/com/taxis99/aws/S3Client.scala]: Codacy Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/99taxis/awsscala/file/2387542317/issues/source?bid=3170898&fileBranchId=3228566#l3)

**Comment by gustavoamigo** [comment]
👍

**Comment by fbenevides** [comment]
👍

**Pull request merged**